internal/nodes/ingress_model_rel.go

```diff
@@ -1182,6 +1182,9 @@ func infraSettingNSToServices(infraSettingName, key string) ([]string, bool) {
 			return allServices, false
 		}
 		for _, svc := range services {
+			if svc.Spec.Type != "LoadBalancer" {
```

Comment:
please use function `isServiceLBType`

Reply:
that function will not be available to other packages (need Upper Camel case)